csunplugged/utils/BaseLoader.py

```diff
@@ -32,6 +36,29 @@ def setup_md_to_html_converter(self):
         custom_processors.add('remove-title')
         self.converter.update_processors(custom_processors)
 
+    def convert_md_file(self, md_file_path):
+        """Returns the Kordac object for a given Markdown file
+
+        Args:
+            file_path: location of md file to convert
+
+        Returns:
+            Kordac result object
+        """
+        content = open(md_file_path, encoding='UTF-8').read()
+        result = self.converter.convert(content)
+        # Write Scratch data for image rendering by Gulp script
+        if 'scratch_images' in result.required_files:
```

Comment:
Should the Scratch render logic be a function outside the `BaseLoader` class, or a method of it? It doesn't require anything within the loader, so is it a function?